Resources/Prototypes/Research/arsenal.yml

```diff
@@ -107,14 +107,15 @@
     state: icon
   discipline: Arsenal
   tier: 1
-  cost: 8000
+  cost: 5000 
   recipeUnlocks:
   - Truncheon
   - TelescopicShield
   - HoloprojectorSecurity
   - WeaponDisablerSMG
   - BorgModuleSecurityDeescalate # Delta-V : SecBorg Research
   - BorgModuleSecurityEscalate # Delta-V : SecBorg Research
+  # Delta-V : cost reduced to 5000 from 8000
```

Comment:
noob do it on the same line instead of this

Reply:
im fairly certain this breaks it but im also dont know for sure

Reply:
It shouldn't.